js/errorNotificationManager.js

```diff
@@ -64,14 +64,14 @@ export default class ErrorNotificationManager extends Backbone.Model {
 
     let isCancellable = true;
 
-    if (config.hasOwnProperty('_isCancellable')) {
+    if (Object.prototype.hasOwnProperty.call(config, '_isCancellable')) {
```

Comment:
```suggestion
    if (Object.hasOwn(config, '_isCancellable')) {
```
`hasOwnProperty` can be called directly on the object so doesn't need this syntax, although we should probably switch to `Object.hasOwn` instead.

Reply:
Unforunately `hasOwn` doesn't seem to be supported by our current version of eslint, but it does work with transpilation, so it's fine to use, even if it seems unavailable in the intellisense dropdowns.